includes/class-admin.php

```diff
@@ -4,58 +4,64 @@ class AffiliateWP_Checkout_Referrals_Admin {
-			'desc' => __( 'Choose how the customer will select the affiliate to award commission to.', 'affiliatewp-checkout-referrals' ),
-			'type' => 'select',
-			'options' => array(
-				'select_menu' => __( 'Customer selects affiliate from list', 'affiliatewp-checkout-referrals'),
-				'input'       => __( 'Customer enters affiliate\'s ID or username', 'affiliatewp-checkout-referrals'),
-			)
-		);
+	public function setting_tab( $tabs ) {
+		$tabs['checkout_referrals'] = __( 'Checkout Referrals', 'affiliatewp-checkout-referrals' );
+		return $tabs;
+	}
 
-		$fields['checkout_referrals_require_affiliate'] = array(
-			'name' => __( 'Require Affiliate Selection', 'affiliatewp-checkout-referrals' ),
-			'desc' => __( 'Customer must select an affiliate to award the referral to', 'affiliatewp-checkout-referrals' ),
-			'type' => 'checkbox'
-		);
+	/**
+	 * Register the settings for Checkout Referrals tab
+	 *
+	 * @since 1.0.9
+	 * @return array
+	 */
+	public function settings( $settings ) {
 
-		$fields['checkout_referrals_affiliate_display'] = array(
-			'name' => __( 'Affiliate Display', 'affiliatewp-checkout-referrals' ),
-			'desc' => __( 'How the affiliates will be displayed at checkout', 'affiliatewp-checkout-referrals' ),
-			'type' => 'radio',
-			'options' => array(
-				'user_nicename' => 'User Nicename',
-				'display_name' => 'Display Name',
-				'nickname'     => 'Nickname'
+		$settings['checkout_referrals'] = array(
+			'checkout_referrals_checkout_text' => array(
+				'name' => __( 'Checkout Text', 'affiliatewp-checkout-referrals' ),
+				'desc' => '<p class="description">' . __( 'Enter the field description to be displayed at checkout.', 'affiliatewp-checkout-referrals' ) . '</p>',
+				'type' => 'text',
+				'std'  => __( 'Who should be awarded commission for this purchase?', 'affiliatewp-checkout-referrals' )
 			),
-			'std' => 'user_nicename'
+			'checkout_referrals_affiliate_selection' => array(
+				'name' => __( 'Affiliate Selection Method', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'Choose how the customer will select the affiliate to award commission to.', 'affiliatewp-checkout-referrals' ),
+				'type' => 'select',
+				'options' => array(
+					'select_menu' => __( 'Customer selects affiliate from list', 'affiliatewp-checkout-referrals'),
+					'input'       => __( 'Customer enters affiliate\'s ID or username', 'affiliatewp-checkout-referrals'),
+				)
+			),
+			'checkout_referrals_require_affiliate' => array(
+				'name' => __( 'Require Affiliate Selection', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'Customer must select an affiliate to award the referral to', 'affiliatewp-checkout-referrals' ),
+				'type' => 'checkbox'
+			),
+			'checkout_referrals_affiliate_display' => array(
+				'name' => __( 'Affiliate Display', 'affiliatewp-checkout-referrals' ),
+				'desc' => __( 'How the affiliates will be displayed at checkout', 'affiliatewp-checkout-referrals' ),
```

Comment:
And a period at the end of this sentence also.
```suggestion
				'desc' => __( 'How the affiliates will be displayed at checkout.', 'affiliatewp-checkout-referrals' ),
```